maker/src/cfd.rs

```diff
@@ -0,0 +1,711 @@
+use crate::collab_settlement;
```

Comment:
I wonder why this isn't marked as moved?

Reply:
That is weird! Maybe the original file wasn't deleted?